externals/CMake/FindNETCDF.cmake

```diff
@@ -1,257 +0,0 @@
-
```

Comment:
We had a lot of duplication in cmake modules.  We don't need two different Findnetcdf.cmake versions.